tests/extend.test.ts

```diff
@@ -19,18 +28,32 @@ describe('Extend', () => {
 
   describe('contract owner', () => {
     let owner: JWKInterface;
+    let walletAddress: string;
+    let emptyWalletCaller: JWKInterface;
 
     beforeAll(async () => {
       owner = getLocalWallet(0);
+      walletAddress = await arweave.wallets.getAddress(owner);
       contract = warp.pst(srcContractId).connect(owner);
+      emptyWalletCaller = getLocalWallet(12);
     });
 
-    it('should not be able to extend a record that is not in its grace period', async () => {
-      const extendYears = MAX_YEARS - 1;
+    afterEach(() => {
+      contract.connect(owner);
+    });
+
+    it('should not be able to extend a record if the caller has insufficient balance', async () => {
+      const extendYears = 1;
       const name = 'name1';
       const { cachedValue: prevCachedValue } = await contract.readState();
       const prevState = prevCachedValue.state as IOState;
       const prevExpiration = prevState.records['name1'].endTimestamp;
+      contract.connect(emptyWalletCaller);
```

Comment:
Just create a new wallet instead of transferring. Use 

`Arweave.crypto.generateJWK()`

Reply:
need to use a wallet that has AR tokens or the transaction wont be executed, resulting in originalTxId being undefined and the appropriate error not being present.